meteor-app/imports/ui/member/add/member-add.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from 'semantic-ui-react'
+import Details from './member-add-details'
+import Emergency from './member-add-emergency'
+import Other from './member-add-other'
+import Steps from './member-add-steps'
+
+class MemberAdd extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      step: 1,
+      formData: {}
+    }
+  }
+
+  onSubmit = ({ formData }) => {
+    this.setState({
+      formData: {
+        ...this.state.formData,
+        ...formData,
+      },
+      step: this.state.step + 1
+    })
+    // if on final step
+    // call props.addMember()
+  }
+
+  onChange = ({ formData }) => {
+    this.setState({
+      formData: {
+        ...this.state.formData,
```

Comment:
I thought you could only use the spread operator once, and it must be the last one?